src/tplink_omada_client/omadasiteclient.py

```diff
@@ -455,3 +455,19 @@ async def get_gateway(self, mac_or_device: Union[str, OmadaDevice, None]) -> Oma
         )
 
         return OmadaGateway(result)
+
+    async def set_led_setting(self, mac_or_device: Union[str, OmadaDevice, None], setting: LedSetting) -> bool:
```

Comment:
Yeah, maybe if we add an `OmadaDeviceDetails` class that inherits from `OmadaDevice` it would be a good option?